pkg/cmd/get/secrets.go

```diff
@@ -42,17 +37,29 @@ var SecretsCmd = &cobra.Command{
 }
 
 // well known secrets that are part of the core packages
-var corePkgSecrets = map[string][]string{
-	"argocd": []string{argoCDInitialAdminSecretName},
-	"gitea":  []string{giteaAdminSecretName},
-}
+var (
+	corePkgSecrets = map[string][]string{
+		"argocd": []string{argoCDInitialAdminSecretName},
+		"gitea":  []string{giteaAdminSecretName},
+	}
+)
 
 type TemplateData struct {
 	Name      string            `json:"name"`
 	Namespace string            `json:"namespace"`
 	Data      map[string]string `json:"data"`
 }
 
+type Secret struct {
+	isCore    bool
```

Comment:
Is the use of `isCore` field just to control the printing of core package fields? Can we do that without this?

Reply:
> Is the use of `isCore` field just to control the printing of core package fields?

It is used to figure out when we have a `core package` secret containing as values: username, password and token (for gitea)

Reply:
> Can we do that without this?

Not really